.github/workflows/check-nixf-tidy.yml

```diff
@@ -0,0 +1,97 @@
+name: Check changed Nix files with nixf-tidy (experimental)
+
+on:
+  pull_request_target:
```

Comment:
Do we really need to run on the `pull_request_target`? Would `pull_request` not also work?

Reply:
GitHub Actions is a fickle beast: https://github.com/NixOS/nixpkgs/blob/d2a2dbc06347b4c5e30dd258743f6b3cae30d985/.github/workflows/check-by-name.yml#L8C1-L8C103